src/screens/EPub/views/EditEPubChapter/ChapterEditor/ChapterContent.js

```diff
@@ -24,7 +25,7 @@ let Tags = ({data, handleDelete}) => {
   }
 
   return (
-    <Box style={boxstyle}>
+    <Box style={boxstyle} data-testid="tag">
```

Comment:
maybe we can rename the test id to include the component name like "ChapterContent-test-tag" in the case we ever have a test with lots of different components?